src/screens/HomeScreen.js

```diff
@@ -173,6 +183,13 @@ export const HomeScreen = ({ navigation, route }) => {
     limitNews = 15,
     limitPointsOfInterestAndTours = 15
   } = sections;
+  const {
+    contentName = 'homeContentList',
+    description: contentListDescription = texts.contentList.description,
+    horizontal = true,
+    showList: showContentList = true,
+    title: contentListTitle = texts.contentList.title
+  } = contentList;
```

Comment:
why are you renaming things here? i suggest having the names you want directly in the configs.

Reply:
Because to avoid confusion in the `homeContentList` content. But I can update this.

Reply:
what do you mean with confusion? where and what confusion?

Reply:
`globalSettings` causes this kind of appearance:
It says `staticContent...` everywhere. I preferred to do it like this to prevent this.
```
    "staticContentList": {
      "horizontal": true,
      "showStaticContentList": true,
      "staticContentListTitle": "Mach Meldung in drei Schritten",
      "staticContentListDescription": "Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.",
      "staticContentName": "homeList"
    }
```
https://github.com/smart-village-solutions/smart-village-app-app/pull/958/commits/c5071d1275e9c9450bb63ab91c3c2de325370162

Reply:
i do not understand. which way do you prefer and why?

Reply:
I changed all the names and made everything the same as in `globalSettings`. But I still think the previous method is better. It will avoid the confusion in `globalSettings`.

after `globalSettings`
```    
"staticContentList": {
      "horizontal": true,
      "showContentList": true,
      "title": "Mach Meldung in drei Schritten",
      "description": "Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.",
      "contentName": "homeList"
    }
```
before `globalSettings`
```
    "staticContentList": {
      "horizontal": true,
      "showStaticContentList": true,
      "staticContentListTitle": "Mach Meldung in drei Schritten",
      "staticContentListDescription": "Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor.",
      "staticContentName": "homeList"
    }
```


Maybe even `showContentList` here can be replaced with just `show`.

If we are already inside a `staticContentList` object, I don't think we need to write `staticContentList...` again, but renaming is necessary to avoid repetition in the code.

Reply:
You prefer shorter configs with more complex code over more descriptive configs with avoiding code logics? ;)
i think no one has problems with configs named precisely which helps in the code. imagine some interface for configs somewhen - nobody will ever see config key names. it will become some json in the background/database. so always keep the focus on lear and maintainable code in first place. with saying this, i clearly prefer named keys we can directly use in code.

Reply:
Yeah, I know that's what you prefer. 😂 That's why I updated the code. Now we leave it as it is, right?